common/helpers

```diff
@@ -43,6 +43,13 @@ list_overlays ()
     } | sort -u
 }
 
+get_cloud_type ()
+{
+   local cloud=`juju show-model| sed -rn 's/.+cloud:\s*(.+).*/\1/p'| uniq`
```

Comment:
`jq` is better than parsing yaml with sed (it can also get unique vales):
```
   local cloud=$(juju show-model --format json |jq -r '[.[].cloud] | unique[]')
   local type=$(juju show-cloud $cloud --format json | jq -r '[.[].type] | unique[]')
```

```

Reply:
Hi pon, yes, jq is better than parsing yaml with sed. Although I merely relocated two original lines without modifying the previous code (perhaps the optimized code should submit another pr), I have now modified to using jq instead of sed to align with your suggestion. thanks for the review.

Reply:
Thanks Joshua. Looks good now!